plugins/kubernetes-backend/src/service/KubernetesProxy.ts

```diff
@@ -36,6 +43,13 @@ export const APPLICATION_JSON: string = 'application/json';
  */
 export const HEADER_KUBERNETES_CLUSTER: string = 'X-Kubernetes-Cluster';
 
+/**
+ * The header that is used to specify the Authentication Authorities token.
+ * e.x if using the google auth provider as your authentication authority then this field would be the google provided bearer token.
+ * @alpha
+ */
+export const HEADER_KUBERNETES_AUTH: string = 'X-Kubernetes-Authorization';
```

Comment:
Noting that prefixing with `X-` is [discouraged](https://www.rfc-editor.org/rfc/rfc6648) and would be better to avoid. It fits with the existing header here ofc so from that PoV the change in this PR is fine. The reason I bring it up is that perhaps we should fix the existing header now that we're making breaking changes anyway.

I would suggest we make these two headers the following:

```
Backstage-Kubernetes-Cluster
Backstage-Kubernetes-Authorization
```

Very much open to suggestions though :grin: CC @backstage/maintainers

Reply:
I agree that the X- prefix has always looked a bit awkward. I have no problems changing it to the header suggestions you provided. I wIll add the changes.

Reply:
Changes added.